apps/nextjs-website/react-components/components/HeroCounter/HeroCounter.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { Box, Typography, Link } from '@mui/material';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+import { isJSX } from '../../types/common/Common.types';
+import ContainerRC from '../common/ContainerRC';
+import { HeroCounterProps } from '@react-components/types/HeroCounter/HeroCounter.types';
+import { BackgroundColor, TextColor } from '../common/Common.helpers';
+import { useTheme } from '@mui/material/styles';
+
+const HeroCounter = (props: HeroCounterProps) => {
+  const {
+    background,
+    theme = 'dark',
+    counterNumber = 0,
+    counterText,
+    title,
+    subtitle,
+    linkText,
+    linkUrl,
+  } = props;
+
+  const backgroundColor = BackgroundColor(theme);
+  const textColor = TextColor(theme);
+
+  const BackgroundImage = isJSX(background) ? (
+    background
+  ) : (
+    <Box
+      role='presentation'
+      sx={{
+        px: 4,
+        position: 'absolute',
+        inset: 0,
+        zIndex: -10,
+        height: '100%',
+        width: '100%',
+        objectFit: 'cover',
+        backgroundSize: 'cover',
+        backgroundPosition: 'center',
+        backgroundImage: `url(${background ?? ''})`,
+      }}
+    />
+  );
+
+  const { palette } = useTheme();
+
+  return (
+    <ContainerRC
+      size='xl'
+      background={!background ? backgroundColor : BackgroundImage}
+      direction='row'
+      sx={{
+        display: 'flex',
+        flexDirection: { xs: 'column', lg: 'row' },
+        alignItems: 'flex-start',
+        justifyContent: 'space-between',
+        py: 4,
+      }}
+    >
+      <Box
+        sx={{
+          flex: 1,
+          display: 'flex',
+          flexDirection: 'column',
+          justifyContent: 'center',
+          alignItems: 'flex-start'
+        }}
+      >
+        <Typography variant='h1' color={textColor} mb={2} sx={{ fontSize: '3.5rem!important' }}>
+          {title}
+        </Typography>
+        <Typography variant='body1' color={textColor} mb={2} sx={{ fontSize: '1rem' }}>
+          {subtitle}
+        </Typography>
```

Comment:
```suggestion
{subtitle && (
        <Typography variant='body1' color={textColor} mb={2} sx={{ fontSize: '1rem' }}>
          {subtitle}
        </Typography>
)}
```
Handle when `subtitle` is undefined.